packages/ag-charts-enterprise/src/features/annotations/toolbar.css

```diff
@@ -0,0 +1,7 @@
+.annotations__text-size {
+    width: 34px;
+    min-width: 0;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    --item-padding: 4px 8px;
+}
```

Comment:
Generally I think it's good to have any CSS variables defined at the top of the ruleset, with a line break below. 

To me it makes it much easier to parse what variables are declared per selector. For example...


```
.annotations__text-size {
     --item-padding: 4px 8px;

     width: 34px;
     min-width: 0;
     border-top-left-radius: 0;
     border-top-right-radius: 0;
 }
 ```

Reply:
I don’t mind going this way, but we should do it in one go because right now they’re always at the bottom